specs/darwin/unified_log.table

```diff
@@ -0,0 +1,16 @@
+table_name("unified_log")
+description("Queries the OSLog framework for entries in the system log.")
+
+schema([
```

Comment:
As these can be search constraints, should they be `index=True`?

Reply:
These look good now with `additional`